Jenkinsfile

```diff
@@ -3,6 +3,13 @@ library changelog: false, identifier: 'lib@master', retriever: modernSCM([
     remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
 ]) _
 
+void runAPItests(String DOCKER_VERSION, OWNER) {
+    stagingJob = build job: 'pmm2-api-tests', parameters: [
+        string(name: 'DOCKER_VERSION', value: DOCKER_VERSION),
```

Comment:
Let's use `DOCKER_IMAGE_VERSION` instead of `DOCKER_VERSION`. It's more accurate.

Reply:
> Let's use `DOCKER_IMAGE_VERSION` instead of `DOCKER_VERSION`. It's more accurate.

+++

Reply:
Or `DOCKER_IMAGE`.

Reply:
Sure, I will update it..and then also update API tests job later, for now just changing the variable name.